src/Symbols/SymbolHelper.cpp

```diff
@@ -36,6 +37,10 @@ using ::orbit_object_utils::CreateObjectFile;
 using ::orbit_object_utils::ElfFile;
 using ::orbit_object_utils::ObjectFile;
 
+constexpr const char* kDeprecationNote =
+    "// !!! Do not remove this comment !!!\n// This file has been deprecated. Please use: Menu > "
+    "Settings > Symbol Locations...\n";
```

Comment:
I think this should mention a version. It needs to be clear to the user that prior to 1.67 (?) this file is still valid and that they shouldn't delete it if they care about older versions of Orbit.

Also I wouldn't say `deprecated`. This doesn't make clear to me that the contents has been migrated. Maybe `This file has been migrated` instead.

Reply:
I changed the wording accordingly. ptal